eth/tracers/native/call.go

```diff
@@ -56,6 +56,29 @@ func (f callFrame) TypeString() string {
 	return f.Type.String()
 }
 
+func (f *callFrame) capture(output []byte, gasUsed uint64, err error) {
+	f.GasUsed = gasUsed
+	output = common.CopyBytes(output)
+	if err == nil {
+		f.Output = output
+		return
+	}
+	f.Error = err.Error()
+	if f.Type == vm.CREATE || f.Type == vm.CREATE2 {
+		f.To = common.Address{}
+	}
```

Comment:
Hm, with `omitempty` set, from https://github.com/ethereum/go-ethereum/pull/25637/files, the idea is that `0x000..` should not be present in the output json. So it's basically `null`. Which is a bit wonky too, but.. Still I wonder why we have this clause there.

Reply:
I think the oriignal difference between outer scope vs inner scope, which was introduced here: https://github.com/ethereum/go-ethereum/commit/8d7e6062ece1550936065aa8a8f8c7aacc584810#diff-c7b3691786d65b22103e27eca0ca1e6650ddf78874b7326bf7afcc30bf6d2b2fR118 -- is because it originally needed to look up the call in a `t.callstack`. And the outermost scope didn't have preceding entry in the callstack. 

As for why the to was set to empty, https://github.com/ethereum/go-ethereum/commit/8d7e6062ece1550936065aa8a8f8c7aacc584810#diff-c7b3691786d65b22103e27eca0ca1e6650ddf78874b7326bf7afcc30bf6d2b2fR131 , I'm not completely sure.

Reply:
So here I just kept what was there for the `CaptureExit` case. Regarding a "null" `to` field, maybe we can have a `To *common.Address` (so a pointer) with `omitempty`, so that we can have trully null values, different from just the `0x00..00` address. WDYT?